### ISSUE TITLE
Folder Selection not working

### ISSUE THREAD

**Issue by mcox86** [mcox86/rpmBIMTools#4]
Some users get the following when selecting the export folder for the export / import schedule dialog open button.

```
System.MissingMethodException: Method not found: '!!0[] System.Array.Empty()'.
   at FolderSelect.FolderSelectDialog.ShowDialog(IntPtr hWndOwner)
   at FolderSelect.FolderSelectDialog.ShowDialog()
   at rpmBIMTools.exportImportSchedules.exportFolderButton_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

Full Error Dump : [error.txt](https://github.com/mcox86/rpmBIMTools/files/407000/error.txt)

**Comment by mcox86** [comment]
Working to catch this exception and falling back to using the new folder selection window api instead, release by end of today.

**Comment by mcox86** [comment]
This issue has been resolved with the latest commit (cbb1adf12138f7475a93b656d6d4ac8307c5d3b7)
